packages/slice-machine/server/tsconfig.json

```diff
@@ -11,6 +11,7 @@
     "noUnusedParameters": true,
     "baseUrl": ".",
     "moduleResolution": "node",
+    "resolveJsonModule": true,
```

Comment:
Why we need that ?

Reply:
package.json is imported on in the server to get the current version and name of the package